web/app/models.py

```diff
@@ -168,14 +169,50 @@ def to_dict(self):
         }
 
 
+RULESETS = ["AGA", "INTERNATIONAL"]
+PAIRINGTYPES = ["McMahon"]
```

Comment:
Can you not define these as normal enums? I admit, my python knowledge is a bit sketchy.